File: Webapi/Webapi/Models/Events/Event.dto.cs
using Webapi.Models.Identity;

namespace Webapi.Models.Events;

#pragma warning disable CS8618 // data transfer object
public class EventDto
{
  public string Id { get; set; }

  public DateTime CreatedAt { get; set; }

  public string Title { get; set; }

  public string? Description { get; set; }

  public DateTime Start { get; set; }

  public DateTime? End { get; set; }

  public EventCategory? Category { get; set; }

  public string? Location { get; set; }

  public string? Notes { get; set; }

  public List<SignUpDto> SignUps { get; set; }
}

File: Frontend/src/app/shared/api/models/event-dto.ts
/* tslint:disable */
/* eslint-disable */
import { EventCategory } from './event-category';
import { SignupDto } from './signup-dto';
export interface EventDto {
  category?: EventCategory;
  createdAt?: string;
  createdBy?: null | string;
  description?: null | string;
  end?: null | string;
  id?: null | string;
  location?: null | string;
  signups?: null | Array<SignupDto>;
  start?: string;
  title?: null | string;
}
